app/models/user.rb

```diff
@@ -77,4 +77,8 @@ def request_admin_status(organisation_id)
   def self.purge_deleted_users_where(query)
     User.deleted.delete_all(query)
   end
+
+  def superadmin_emails
```

Comment:
Maroo-b, so `superadmin_emails` should be a class method instead of an instance method. Can you explain why? I did notice we had an `edit_org_waiting_for_approval` method but I got confused because in his notes Sam referred to `deliver_edit_org_waiting_for_approval`. Maybe a typo but it looks like you are right. Once we determine how to proceed, I will work on the tests.